proco/schools/api.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import mixins, viewsets
+
+from django_filters.rest_framework import DjangoFilterBackend
+
+from proco.schools.models import School
+from proco.schools.serializers import SchoolSerializer
+
+
+class SchoolsViewSet(
```

Comment:
please add test for not authorized user. i believe IsAuthenticated is used by default